data_validation/__main__.py

```diff
@@ -388,9 +388,24 @@ def config_runner(args):
                 )
             config_file_names = cli_tools.list_validations(config_dir=args.config_dir)
             config_managers = []
+            errors = False
             for file in config_file_names:
                 config_managers = build_config_managers_from_yaml(args, file)
-                run_validations(args, config_managers)
+                try:
+                    logging.info(
+                        "Currently running the validation for YAML file: %s",
+                        file,
+                    )
+                    run_validations(args, config_managers)
+                except Exception as e:
+                    errors = True
+                    logging.error(
+                        "Error %s occurred while running config file %s. Skipping it for now.",
```

Comment:
```suggestion
                        "Error '%s' occurred while running config file %s. Skipping it for now.",
```
To make it clearer what part of the string is the exactly error message returned.

Reply:
Thank you